internal/provider/models/processors/test/route_test.go

```diff
@@ -421,13 +421,13 @@ func TestAccRouteProcessor(t *testing.T) {
 						title = "blackhole title"
 						description = "blackhole desc"
 						pipeline_id = mezmo_pipeline.test_parent.id
-						inputs = [mezmo_route_processor.with_outputs.conditionals.0.output_name]
+						inputs = [tolist(mezmo_route_processor.with_outputs.conditionals).0.output_name]
```

Comment:
There's code smell here.  We shouldn't have to do this and we should figure out why.  This also means we'd have to fix the examples.  Let's look into this more.

Reply:
Sounds like you can just rebase and drop this commit after our findings.